My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
```

Comment:
Note: To package your custom extensions to kernel classes with your project, leaving the kernel clean, put methods in protocol *MyPackageName (yes, it is a hack, but that's the way it's done!)